configs/raspberrypi3_wpe_rdk_defconfig

```diff
@@ -23,6 +23,9 @@ BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,865ddc
 BR2_LINUX_KERNEL_DEFCONFIG="bcm2709"
 BR2_LINUX_KERNEL_DTS_SUPPORT=y
 BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2710-rpi-3-b bcm2710-rpi-3-b-plus bcm2710-rpi-cm3"
+BR2_PACKAGE_CONNMAN=y
+BR2_PACKAGE_CONNMAN_WIFI=y
+BR2_PACKAGE_CONNMAN_CLIENT=y
```

Comment:
What do you think about keeping this defconfig as minimal as possible, and adding a new `raspberrypi3_wpe_wiki_defconfig` which adds the WiFi support? I think it would be good to keep a minimal configuration that people can use as basis for their own.

Reply:
Make sense. The only reason I added this because My PC is far from router and I don't have long Ethernet cable to connect so I am using wifi :). Anyway I will revert this file and may be later add another defconfig with wifi support.

Reply:
Done. I will add new defconfig with wifi support next week.